converter.py

```diff
@@ -0,0 +1,21 @@
+import csv
+from models.arshin import Application
+from xsdata.formats.dataclass.serializers import XmlSerializer
+import factory
+
+
+def CsvToXmlConverter(input_filename, output_filename='application'):
+    application = Application()
+
+    with open(input_filename, 'r', encoding='Windows-1251') as file:
+        reader = csv.reader(file, delimiter=';')
+        for row in reader:
+            if all(string == '' for string in row):
```

Comment:
📝 This validation could be done either in the factory by throwing an exception on in a separate input validation block.